src/components/forms/JurisdictionSelect/index.tsx

```diff
@@ -208,6 +210,26 @@ export const promiseOptions = (
       });
   });
 
+/**
+ * Sets value in name and title field
+ * @param e Form event
+ * @param form Formik form Object
+ */
+export const onSelectionComplete = (e: FormEvent<Element>, form: FormikProps<any>) => {
+  const target = e.target as HTMLInputElement;
+  const fieldsThatChangePlanTitle = ['interventionType', 'fiStatus', 'date', 'jurisdictions[0].id'];
```

Comment:
Is there a way to have this code shared between this module and the [similar thing](https://github.com/onaio/reveal-frontend/blob/08d57148cc60210d60c5b10718970bd5e508a372/src/components/forms/PlanForm/index.tsx#L397) in the PlanForm component?